models/public/mobilenet-v2-pytorch/mobilenet-v2-pytorch.md

```diff
@@ -0,0 +1,80 @@
+# mobilenet-v2-pytorch
+
+## Use Case and High-Level Description
+
+MobileNet V2 is image classification model pretrained on ImageNet dataset. This
+is a PyTorch implementation of MobileNetV2 architecture as described in
+the paper ["Inverted Residuals and Linear Bottlenecks: Mobile Networks for Classification,
+Detection and Segmentation"](https://arxiv.org/pdf/1801.04381.pdf).
+
+The model input is a blob that consists of a single image of "1x3x224x224"
+in RGB order.
+
+The model output is typical object classifier for the 1000 different classifications
+matching with those in the ImageNet database.
+
+## Example
+
+See [here](https://github.com/tonylins/pytorch-mobilenet-v2)
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Classification|
+| GFLOPs            | 0.615         |
+| MParams           | 3.489         |
+| Source framework  | PyTorch\*     |
+
+## Accuracy
+
+| Metric | Original model | Converted model |
+|--------|----------------|-----------------|
+| Top 1  | 71.8%          | 71.8%           |
+
+## Performance
+
+## Input
+
+### Original model
+
+Image, name - `data`,  shape - `1,3,224,224`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `RGB`.
+Mean values - [123.675,116.28,103.53], scale value - [58.624,57.12,57.375]
+
+### Converted model
+
+Image, name - `data`,  shape - `1,3,224,224`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `RGB`
```

Comment:
Wouldn't it make sense to reverse the channels during conversion, given that we already do that for most (possibly all) other models?

Reply:
Because we reverse channels only if they are in `BGR` order. So, all public converted models have `RGB` input. This is a convention for public models, as I know.

Reply:
Are you sure about this? I randomly opened a few models, and all of them have BGR input in the converted form (either because they already had it originally, or because the channels were reversed during conversion).